full-service/src/service/wallet.rs

```diff
@@ -941,7 +939,7 @@ mod tests {
         });
         let result = dispatch(&client, body, &logger);
         let balance_status = result.get("status").unwrap();
-        let unspent = balance_status.get(TXO_UNSPENT).unwrap().as_str().unwrap();
+        let unspent = balance_status.get("unspent").unwrap().as_str().unwrap();
```

Comment:
Here and in other places, the intent was to get a field from the JSON response object, but the code incorrectly used the database constants.

Reply:
(In general, it would probably be easier to avoid get("foo") altogether, and instead use serde to convert the response JSON into the response struct, and assert that the deserialized struct equals the expected response struct.)

Reply:
Sounds great, ty!